### PR TITLE
[MODORDERSTO-412] - Create Karate tests for Open/UnOpen/Change Instance in cross-tenant envs

### PR DESCRIPTION
**Pull request by azizbekxm**
## Purpose
https://folio-org.atlassian.net/browse/MODORDSTOR-412

## Approach
karate tests:
- Fix consortia order structure to compatible with cucumber report
- Fix open-order feature to compatible with best practices
- add test when we create instance in PO Line using free-text and open order https://folio-org.atlassian.net/browse/MODORDERS-1129 and related PR https://github.com/folio-org/mod-orders/pull/963 . In this case our code should create new instance in central tenant and share it in member tenant before creating holding and item in member tenant
- add test when we have instance already created in central tenant, but do not have in member tenant and we open order. In this case our code should share instance to member tenant and create Inventory in member tenant (the same as previous PR)
- add test when we have instance in both central and member tenant already shared and we open order. In this case our code should not create new instances, just create Inventory in correct tenant. (the same PR as previous)
- add test for closing order and check that related item status also is changed from On order to Order closed , https://folio-org.atlassian.net/browse/MODORDERS-1105 and related PR - https://github.com/folio-org/mod-orders/pull/931/files

### CODE REVIEW

File: acquisitions/src/main/resources/thunderjet/consortia/consortia-orders-junit.feature [hunk 1]
```diff
@@ -72,17 +74,30 @@ Feature: mod-consortia integration tests
...
+    * call read('order-utils/finances.feature')
+    * call read('order-utils/organizations.feature')
+    * call read('order-utils/orders.feature')
+
   Scenario: Create and open order
-    * call read('consortia-orders.feature')
+    Given call read('features/open-order-with-locations-from-different-tenants.feature')
+
+#  Scenario: Test cross-tenant inventory objects creation when working with pieces
+#    Given call read("features/pieces-api-test-for-cross-tenant-envs.feature")
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/consortia/consortia-orders-junit.feature:100]: is this commented by mistake?

Comment (azizbekxm) [reply acquisitions/src/main/resources/thunderjet/consortia/consortia-orders-junit.feature]: thanks to point out. reverted back changes

File: acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature [hunk 2]
```diff
@@ -14,76 +13,856 @@ Feature: Open ongoing order
...
+    And match $.totalRecords == 2
+
+    Given path 'inventory/items'
+    And header x-okapi-tenant = universityTenant
+    And param query = 'purchaseOrderLineIdentifier==' + poLineId
+    When method GET
+    And match $.totalRecords == 2
+
+
+    ## 6. Update Order to close
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature:415]: very good scenarios

File: acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature [hunk 3]
```diff
@@ -14,76 +13,856 @@ Feature: Open ongoing order
...
+
+    # Update Order to Open
+    Given path 'orders/composite-orders', orderId
+    And param deleteHoldings = true
+    And request orderResponse
+    When method PUT
+    Then status 204
+
+
+    ## Verify Inventory 'Instnace, Holding, Item' existance in member tenant
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature:585]: small typo, should be centralTenant in comment

File: acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature [hunk 4]
```diff
@@ -14,76 +13,856 @@ Feature: Open ongoing order
...
+    # 3.4 Check items in 'centralTenant'
+    Given path 'inventory/items'
+    And param query = 'purchaseOrderLineIdentifier==' + poLineId
+    And header x-okapi-tenant = centralTenant
+    When method GET
+    And match $.totalRecords == 2
+    * def centralPoLineItemId1 = response.items[0].id
+    * def centralPoLineItemId2 = response.items[1].id
+
+    ## 4. Verify Shadow Instance, Holding, Item in 'centralTenant'
```
Comment (SerhiiNosko) [inline acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature:700]: comment should be updated to memberTenant

Comment (azizbekxm) [reply acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature]: Thanks, updated

File: acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature (hunk 3 above)
Comment (azizbekxm) [reply acquisitions/src/main/resources/thunderjet/consortia/features/open-order-with-locations-from-different-tenants.feature]: updated

**Review by SerhiiNosko — APPROVED** [review]

**Review by imerabishvili — APPROVED** [review]

**Comment by azizbekxm** [comment]
image

**Pull request merged**